documentation/bolt.md

```diff
@@ -85,4 +91,6 @@ Bolt is an open source orchestration tool that automates the manual work it take
 </table>
 
 
-To send us feedback or let us know about a docs error, [open a ticket](https://tickets.puppetlabs.com/browse/DOCUMENT/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel) (you'll need a Jira account) or [email the docs team](mailto:docs@puppet.com).
\ No newline at end of file
+To send us feedback or let us know about a docs error, [open a
+ticket](https://tickets.puppetlabs.com/browse/DOCUMENT/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel)
+(you'll need a Jira account) or [email the docs team](mailto:docs@puppet.com).
```

Comment:
Should this be updated to point at Github Issues instead?

Reply:
Good point... I can change this, but there is a snippet that the web build adds to the bottom of each page that also links to Jira. I'll talk to the web team about changing that for the Bolt docs. I think we should try do that around the same time though so there isn't conflicting information.